pkg/asset-manager-utils/contracts/aave/IAaveIncentivesController.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: agpl-3.0
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+interface IAaveIncentivesController {
+    function handleAction(
+        address asset,
+        uint256 userBalance,
+        uint256 totalSupply
+    ) external;
+
+    function getRewardsBalance(address[] calldata assets, address user) external view returns (uint256);
+
+    function claimRewards(
```

Comment:
Weird, these functions are not in this interface in version 1.0.1 of the protocol-v2 package. Where did you retrieve them from?

Reply:
I pulled these from the verified contracts on Etherscan for the rewards contract. Strangely looking back I can't re-find these so we should drop the final param of `claimRewards`.

We should drop the other two functions as they are unused.

Reply:
Let's double check with the Aave team that these are the ones we should be using then.